app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :email, presence: true
+  validates :password, presence: true
```

Comment:
- [ ] [OPTIONAL] Go a step further in your validations, in real projects you'll also validate the formats of the fields, not just their presence:
  - Can you validate the min and max length of the username? What about forbidden characters?
  - Can you check the the email has a valid email format?
  - Can you check the min and max size of the password and the required types of characters it should have?